pkg/csi/lib/parser.go

```diff
@@ -407,6 +417,33 @@ func ParseVG(line string) (*VG, error) {
 	}, nil
 }
 
+// ParsePV parses physical volume
+func ParsePV(line string) (*PV, error) {
+	// pvs --units=b --separator="<:SEP:>" --nosuffix --noheadings -o pv_name,pv_size,pv_free,pv_uuid,pv_tags,vg_name -S vg_name=<vg> --nameprefixes -a
+	fields, err := parse(line, 6)
+	if err != nil {
+		return nil, err
+	}
+
+	size, err := strconv.ParseUint(fields["LVM2_PV_SIZE"], 10, 64)
```

Comment:
Those can also be expressed as a constant: 
- LVM2_PV_NAME
- LVM2_PV_SIZE
- LVM2_PV_UUID
- LVM2_PV_TAGS